src/SFA.DAS.CloudService/ServiceConfiguration.Release.cscfg

```diff
@@ -5,10 +5,10 @@
     <ConfigurationSettings>
       <Setting name="Microsoft.WindowsAzure.Plugins.Diagnostics.ConnectionString" value="__StorageConnectionString__" />
       <Setting name="EnvironmentName" value="__EnvironmentName__" />
-      <Setting name="StorageConnectionString" value="__StorageConnectionString__" />
+      <Setting name="StorageConnectionString" value="__ConfigurationStorageConnectionString__" />
       <Setting name="ConfigurationStorageConnectionString" value="__ConfigurationStorageConnectionString__" />
```

Comment:
Nope that's wrong.
A copy past miss from **ServiceConfiguration.Cloud.cscfg**
Should I change it in that file as well?